### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by julia-rabello**
**What problem is this solving?**

Reviewed the app documentation

**How should this be manually tested?**

**Screenshots or example usage:**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,86 +6,140 @@
...
-How to integrate your app with **Storefront Permissions** and make it available within the Role management
+| **Role** | **Key** | **Description** |
+|---|---|---|
+| Store Admin | `store-admin` | Store administrator, that is, a user who has access to the VTEX Admin. |
+| Sales Admin | `sales-admin` | Sales administrator user who can manage all sales users. |
+| Sales Manager | `sales-manager` | Sales manager user who can manage sales users in the same organization, as well as assist or impersonate buyers during navigation or purchase. |
+| Sales Representative | `sales-representative` | Sales representative user who can assist or impersonate buyers during navigation or purchase. |
+| Organization Admin | `customer-admin` | Main organization user who manages the organization information, as well as its members and cost centers. |
+| Organization Approver | `customer-approver` | Organization user who is in charge of reviewing orders placed by buyers, with the ability to approve them or deny them. |
+| Organization Buyer | `customer-buyer` | Organization user who has the ability to place orders in your store. |
```
Comment (polishq) [inline docs/README.md:30]: The information about `Approver` and `Buyer` is not accurate. Despite their name, by default the `Buyer` cannot place orders. They are able to add items to cart, and, if the B2B Quotes app is installed, they are able to save their cart as a saved cart or quote.

`Approvers` then are the ones who can take a saved cart (or quote) that was created by a `Buyer` and use it to place an order.

File: docs/README.md [hunk 2]
```diff
@@ -6,86 +6,140 @@
...
-## Advanced app integration
+In the following table, you can see the available storefront roles, their key used for identification in the app’s code, and their description.
 
-How to integrate your app with **Storefront Permissions** and make it available within the Role management
+| **Role** | **Key** | **Description** |
+|---|---|---|
+| Store Admin | `store-admin` | Store administrator, that is, a user who has access to the VTEX Admin. |
+| Sales Admin | `sales-admin` | Sales administrator user who can manage all sales users. |
+| Sales Manager | `sales-manager` | Sales manager user who can manage sales users in the same organization, as well as assist or impersonate buyers during navigation or purchase. |
+| Sales Representative | `sales-representative` | Sales representative user who can assist or impersonate buyers during navigation or purchase. |
```
Comment (polishq) [inline docs/README.md:27]:
```suggestion
| Sales Representative | `sales-representative` | Sales representative user who can assist or impersonate buyers in the same cost center during navigation or purchase. |
```

**Review by polishq — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (julia-rabello) [reply docs/README.md]: I've changed it to the following:

| | | |
|---|---|---|
| Organization Approver | `customer-approver` | Organization user who can take a saved cart or quote that was created by an **Organization Buyer** and use it to place an order. |
| Organization Buyer | `customer-buyer` | Organization user who has the ability to to add items to cart. If the **B2B Quotes** app is installed, they are also able to save their cart for future use or create a quote. |

File: docs/README.md [hunk 3]
```diff
@@ -6,86 +6,140 @@
...
+| **Role** | **Key** | **Description** |
+|---|---|---|
+| Store Admin | `store-admin` | Store administrator, that is, a user who has access to the VTEX Admin. |
+| Sales Admin | `sales-admin` | Sales administrator user who can manage all sales users. |
+| Sales Manager | `sales-manager` | Sales manager user who can manage sales users in the same organization, as well as assist or impersonate buyers during navigation or purchase. |
+| Sales Representative | `sales-representative` | Sales representative user who can assist or impersonate buyers in the same cost center during navigation or purchase. |
+| Organization Admin | `customer-admin` | Main organization user who manages the organization information, as well as its members and cost centers. |
+| Organization Approver | `customer-approver` | Organization user who can take a saved cart or quote that was created by an **Organization Buyer** and use it to place an order. |
+| Organization Buyer | `customer-buyer` | Organization user who has the ability to to add items to cart. If the **B2B Quotes** app is installed, they are also able to save their cart for future use or create a quote. |
```
Comment (polishq) [inline docs/README.md:30]:
```suggestion
| Organization Buyer | `customer-buyer` | Organization user who has the ability to add items to cart. If the **B2B Quotes** app is installed, they are also able to save their cart for future use or create a quote. |
```

**Review by polishq — CHANGES_REQUESTED** [review]

**Review by polishq — APPROVED** [review]
LGTM! Thanks for helping with this

**Pull request merged**